cf-custom-resources/lib/custom-domain-app-runner.js

```diff
@@ -0,0 +1,189 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/* jshint node: true */
+/*jshint esversion: 8 */
+
+"use strict";
+
+const AWS = require('aws-sdk');
+const { report, defaultSleep } = require('../lib/partials');
```

Comment:
Gotcha.

I will copy-paste as well since we probably don't need to `zipAndUpload` this.

The `aws-sdk` package can be `zipAndUpload`ed and created as a lambda layer. That way the lambda function itself can be uploaded as usual.

Although `partials` can be `zipAndUpload`ed and created as a layer too (or add to the `aws-sdk` layer), it'll need to be copy-pasted to here anyways when lambda starts using the new `aws-sdk` version (so that we won't need to create the lambda layer)